examples/COCO/README.md

```diff
@@ -0,0 +1,38 @@
+## Unet on Coco segmentation task
+
+Trains Unet on the COCO dataset.
+
+Adopted from https://github.com/google-research/scenic for Unet and 
+https://github.com/baudcode/tf-semantic-segmentation for COCO dataset, 
+we add the following modifications to better demonstrate the training speedup by Alpa. And we could also show Alpa's scalibility here.
+- We allow the number of convolution layers at each height to be changed which are 2 for default unet configuration, making it more flexible to scale up. 
+- Use a large batch size.
+- ...
+
+### Requirements
+* Tensorflow 2.8.0
+* imageio 2.16.0
+* opencv-python 4.5.5.62
+
+### Scalability 
+
+| # of GPU   | 2 | 4 | 8 | 16 | 32 |
+| :------ | -----: | -------: | -------------: | ----------: | :---------------------------------------- |
+| config(bsz, channelsz, blockcnt): |  (256, (64, 64, 96, 128), (6, 6, 6, 6, 6)) | (256, (64, 128, 256, 256), (10, 10, 10, 10, 10)) | (64, (384, 384), (32, 64, 128, 256), (8, 8, 8, 8, 8)) | | |
+| compilation time(s): | 155.21 | 6249.41 | 805.98 | | |
+| one batch execution time(s): | 16.63 | 19.27 | 1.96 | | |
```

Comment:
The result has not been finished yet, because I have successfully set up all configurations. As I said, when number of gpus and models size scaling up, unet could not be parallelized on my server. 

For the four questions
- Yes, time indeed increases a lot and increas both in terms of compilation and profiling. The following is more detailed statistics. 
![image](https://user-images.githubusercontent.com/45677459/158005361-30e19ff0-7c2f-4513-b99a-0dfd0f8e76e3.png)
![image](https://user-images.githubusercontent.com/45677459/158005368-a2094e6b-679b-464a-90d8-e0a3ba0140f4.png)

- I want to first fully utilize the memory to see how large the model size could go. After alpa could run smoothly in my server, I will run them again. Now this statistics could not be used to represent Alpa's scalability. 
- Because now I could not train large model on 8 GPUs successfully. 
- Yes, it is weird, especially when you look at 2GPU and 4 GPU cases together.